src/gamecontroller.cs

```diff
@@ -2170,5 +2202,485 @@ interface GCProductCategory
 
 		[Field ("GCProductCategoryKeyboard")]
 		NSString Keyboard { get; }
+
+		[iOS (16,0), Mac (13,0), NoWatch, TV (16,0), MacCatalyst (16,0)]
+		[Field ("GCProductCategoryHID")]
+		NSString GCProductCategoryHid { get; }
+	}
+
+	[iOS (16,0), Mac (13,0), NoWatch, TV (16,0), MacCatalyst (16,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface GCRacingWheel : GCDevice {
+		[Static]
+		[Export ("connectedRacingWheels")]
+		NSSet<GCRacingWheel> ConnectedRacingWheels { get; }
+
+		[Export ("acquireDeviceWithError:")]
+		bool AcquireDevice ([NullAllowed] out NSError error);
+
+		[Export ("relinquishDevice")]
+		void RelinquishDevice ();
+
+		[Export ("acquired")]
+		bool Acquired { [Bind ("isAcquired")] get; }
+
+		// [Export ("wheelInput", ArgumentSemantic.Strong)]
+		// GCRacingWheelInput WheelInput { get; }
+
```

Comment:
Why is this commented out?

Reply:
It returns the type of GCRacingWheelInput which is commented out due to not building. I posted the question from xcode14 channel in the other comment